packages/admin/src/table/filterbar/FilterBarMoreFilterButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { makeStyles, Theme, Typography } from "@material-ui/core";
+import { MoreHoriz } from "@material-ui/icons";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface StyleProps {
+    fieldBarWidth: number;
+}
+
+const useStyles = makeStyles(
+    (theme: Theme) => ({
+        showMoreWrapper: {
+            minWidth: (props: StyleProps) => `${props.fieldBarWidth}px`,
+            border: `1px solid ${theme.palette.grey[300]}`,
+            justifyContent: "center",
+            position: "relative",
+            marginBottom: "10px",
+            alignItems: "center",
+            marginRight: "10px",
+            cursor: "pointer",
+            display: "flex",
+        },
+        showMoreTextWrapper: {
+            marginLeft: "15px",
+        },
+    }),
+    { name: "CometAdminFilterBar" },
+);
+
+export interface FilterBarMoreFilterButtonProps {
+    fieldBarWidth: number;
+}
+
+export function FilterBarMoreFilterButton({ fieldBarWidth, children }: React.PropsWithChildren<FilterBarMoreFilterButtonProps>) {
```

Comment:
This is actually more than a button, and I was confused when it returned the `children` when extended. Therefore, I'd call this `FilterBarMoreFilters` or `FilterBarAdditionalFilters`